client/my-sites/upgrades/domain-management/name-servers/index.jsx

```diff
@@ -11,7 +11,7 @@ import Main from 'components/main';
 import Header from 'my-sites/upgrades/domain-management/components/header';
 import CustomNameserversForm from './custom-nameservers-form';
 import WpcomNameserversToggle from './wpcom-nameservers-toggle';
-import IcannVerificationCard from './icann-verification-card';
+import IcannVerificationCard from '../components/icann-verification-card';
```

Comment:
You should use absolute paths if the file you're importing is not in the same folder.